scripts/abi_check.py

```diff
@@ -231,17 +256,37 @@ def run_main():
             help="keep all reports, even if there are no compatibility issues",
         )
         parser.add_argument(
-            "-o", "--old-rev", type=str, help="revision for old version",
-            required=True
+            "-o", "--old-rev", type=str,
+            help=("revision for old version."
+                  "Can include repository before revision"),
+            required=True, nargs="+"
         )
         parser.add_argument(
-            "-n", "--new-rev", type=str, help="revision for new version",
-            required=True
+            "-n", "--new-rev", type=str,
+            help=("revision for new version"
+                  "Can include repository before revision"),
+            required=True, nargs="+"
         )
         abi_args = parser.parse_args()
+        if len(abi_args.old_rev) == 1:
+            old_repo = None
+            old_rev = abi_args.old_rev[0]
+        elif len(abi_args.old_rev) == 2:
+            old_repo = abi_args.old_rev[0]
+            old_rev = abi_args.old_rev[1]
```

Comment:
Would it be easier to understand and more readable to have "--old-rev" and "--old-repo" options instead of using repeated "--old-rev" to specify the repo? At least users wouldn't have to remember which order to use.

Reply:
You're probably right. There's a lot of options already, a few more won't hurt.